components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/Utils.java

```diff
@@ -449,4 +463,161 @@ public static String addDomainToName(String name, String domainName) {
+
+        String carbonHomePath = Utils.getCarbonHome();
+        String resolvedPath = formatPath(carbonHomePath + File.separator + registryPath + File.separator);
+        try {
+            File file = new File(resolvedPath);
+            return file.exists();
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    /**
+     * Format the path by adding the relevant registry type prefix.
+     * @param path  Registry path
+     * @return      Formatted path with the prefix
+     */
+    public static String getRegistryPathPrefix(String path) {
+
+        String pathWithPrefix;
+        if (path.startsWith(CONFIGURATION_REGISTRY_PATH)) {
+            pathWithPrefix = path.replace(CONFIGURATION_REGISTRY_PATH, CONFIGURATION_REGISTRY_PREFIX);
+        } else if (path.startsWith(GOVERNANCE_REGISTRY_PATH)) {
+            pathWithPrefix = path.replace(GOVERNANCE_REGISTRY_PATH, GOVERNANCE_REGISTRY_PREFIX);
+        } else if (path.startsWith(LOCAL_REGISTRY_PATH)) {
+            pathWithPrefix = path.replace(LOCAL_REGISTRY_PATH, LOCAL_REGISTRY_PREFIX);
+        } else {
+            return null;
+        }
+        return pathWithPrefix;
+    }
+
+    /**
+     * This method returns a string containing file content.
+     *
+     * @param axis2MessageContext Axis2 message context
+     * @return A string containing file content
+     */
+    public static String getPayload(org.apache.axis2.context.MessageContext axis2MessageContext) {
+
+        try {
+            InputStream inputStream = getInputStream(axis2MessageContext, true);
+            return IOUtils.toString(inputStream);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    private static InputStream getInputStream(org.apache.axis2.context.MessageContext messageContext, boolean reset) {
+        if (messageContext == null) {
+            return null;
+        } else {
+            Object o = messageContext.getProperty("bufferedInputStream");
+            if (o instanceof InputStream) {
+                InputStream is = (InputStream) o;
+                if (reset && is.markSupported()) {
+                    try {
+                        is.reset();
+                    } catch (IOException var5) {
+                        return null;
+                    }
+                }
+                return is;
+            } else {
+                return null;
+            }
+        }
+    }
+
+    /**
+     * This method returns a byte array containing file content.
+     *
+     * @param messageContext    Synapse message context
+     * @return                  A byte array containing file content
+     */
+    public static byte[] getPayloadFromMultipart(MessageContext messageContext) {
+
+        try {
+            OMNode fileOmNode = messageContext.getEnvelope().getBody().getFirstElement()
+                    .getFirstChildWithName(new QName("file")).getFirstOMChild();
```

Comment:
is "file" reserved keyword ? if so pls use a constant

Reply:
Changed it with a constant